app/views/agent/TaxYears.scala.html

```diff
@@ -0,0 +1,63 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.templates.agent.MainTemplate
+@import views.html.helpers.injected.BackLink
+
+@this(mainTemplate: MainTemplate, backLink: BackLink)
+
+@(taxYears: Seq[Int], backUrl: String, itsaSubmissionIntegrationEnabled: Boolean)(implicit request: Request[_], messages: Messages, appConfig: config.FrontendAppConfig)
+
+@taxYearRow(year: Int) = {
+	<div class="govuk-summary-list__row">
+		<dd class="govuk-summary-list__value">
+			@messages("taxYears.taxYears", s"${year - 1}", s"$year")
+		</dd>
+		@if(itsaSubmissionIntegrationEnabled) {
+			<dd class="govuk-summary-list__actions">
+				<a class="govuk-link" id="updateReturn-link-@year" href="@appConfig.submissionFrontendUrl">
+					@messages("taxYears.updateReturn")
+					<span class="visuallyhidden">@messages("taxYears.taxYears", s"${year - 1}", s"$year")</span>
+				</a>
+			</dd>
+		} else {
+		}
```

Comment:
Optional cleanup suggestion: remove unneeded `else` clause
```suggestion
		} 
```

Reply:
Remove the else